projects/spring-security/HexFuzzer.java

```diff
@@ -0,0 +1,22 @@
+import com.code_intelligence.jazzer.api.FuzzerSecurityIssueHigh;
+
+import java.lang.CharSequence;
+
+import org.springframework.security.crypto.codec.Hex;
+
+public class HexFuzzer {
+    public static void fuzzerTestOneInput(byte[] data) {
+        final byte[] initialByteArray = data;
+        final char[] encodedChars;
+
+        try {
+            encodedChars = Hex.encode(initialByteArray);
+
+            if (! initialByteArray.toString().equals(Hex.decode(encodedChars.toString()))) {
+                throw new FuzzerSecurityIssueHigh("Hex value has changed during encoding and decoding");
```

Comment:
`FuzzerSecurityIssueHigh` is probably a bit too much, I would set to low or medium